Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerComponentViewModel.cs

```diff
@@ -103,8 +103,11 @@ private void SetName()
                     case ComponentType.Document:
                         using (var app = _vbe?.HostApplication())
                         {
-                            var parenthesized = app?.GetDocument(qualifiedModuleName)?.DocumentName ?? string.Empty;
-                            _name = string.IsNullOrEmpty(parenthesized) ? _name : $"{_name} ({parenthesized})";
+                            if (app!=null && app.TryGetDocument(qualifiedModuleName, out var document))
```

Comment:
Spaces in `app != null`? My brain really wants that to be a syntax error until I stare at it.